petabtests/cases/0017/0017.md

```diff
@@ -0,0 +1,18 @@
+# PEtab test case 0017
+
+## Objective 
+
+This case tests support for partial preequilibration with `NaN`'s in the 
+condition file.
+
+The model is to be simulated for a preequilibration condition and a
+simulation condition.
+For preequilibration, species `B` is initialized with `0`. For simulation,
+`B` is set to `NaN`, meaning that it is initialized with the result from
+preequilibration.
+For `A` the preequilibration result is to be used.
```

Comment:
What about changing the test to reinitializing A (and continuing B) to ensure, that not just everything is continued from steadystate if there is a NaN somewhere? Looks good otherwise, thanks.

Reply:
Sounds good. I changed it accordingly.